.github/actions/install-go/action.yml

```diff
@@ -0,0 +1,16 @@
+name: "Setup Go"
+description: "Reusable action to install Go, so there is one place to bump Go versions"
+inputs:
+  go-version:
+    required: true
+    default: "1.21.6"
```

Comment:
Can we have the go version defined in a file inside the repository and then be used in actions as well as other places? Similar to how we have [runc](https://github.com/containerd/containerd/blob/main/script/setup/runc-version) and [critools](https://github.com/containerd/containerd/blob/main/script/setup/critools-version) version defined. This will also help to make the same go version to be used in the [release](https://github.com/containerd/containerd/blob/main/.github/workflows/release.yml#L121) workflow where we pass the version as `--build-arg` instead of using `setup-go`

Reply:
Yeah, that's possible. I can take a look and make a follow up PR.
This suggestion is much wider scope of work.